app/scripts/util/formatting/date-format.js

```diff
@@ -22,27 +49,19 @@ const DateFormat = {
             dt = new Date(dt);
         }
         return dt
-            ? dt.getDate() + ' ' + Locale.monthsShort[dt.getMonth()] + ' ' + dt.getFullYear()
+            ? new Intl.DateTimeFormat(SettingsManager.activeLocale, {
+                  year: 'numeric',
+                  month: 'short',
+                  day: 'numeric'
+              }).format(dt)
             : '';
     },
 
     dtStrFs(dt) {
         if (typeof dt === 'number') {
             dt = new Date(dt);
         }
-        return dt
-            ? dt.getFullYear() +
-                  '-' +
-                  StringFormat.pad(dt.getMonth() + 1, 2) +
-                  '-' +
-                  StringFormat.pad(dt.getDate(), 2) +
-                  'T' +
-                  StringFormat.pad(dt.getHours(), 2) +
-                  '-' +
-                  StringFormat.pad(dt.getMinutes(), 2) +
-                  '-' +
-                  StringFormat.pad(dt.getSeconds(), 2)
-            : '';
+        return dt ? new Date().toISOString().replaceAll(':', '-').slice(0, 19) : '';
```

Comment:
This will always print the current date if I'm not mistaken. There are tests for this module, they also need an update I guess.

Reply:
If a method accepts date as a parameter, it should print exactly this date, not the current date. I mean, look, you do `new Date()` if `dt` is passed, that's the mistake, you meant `dt` here right? Something like `return dt ? dt.toString... : ''`

Reply:
Yes @antelle, i have **already fixed that** in the next commit. I didn't notice it before :D